google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1153,6 +1159,19 @@ public Builder setAsyncExecutorProvider(CloseableExecutorProvider provider) {
       return this;
     }
 
+    /**
+     * Sets the {@link DirectedReadOption} that specify which replicas or regions should be used for
+     * non-transactional reads or queries.
+     *
+     * <p>DirectedReadOptions set at the request level will take precedence over the options set
+     * using this method.
+     */
+    public Builder setDirectedReadOption(DirectedReadOptions directedReadOptions) {
```

Comment:
nit: `setDirectedReadOption` -> `setDirectedReadOptions`?

Please feel free to retain as is, if you feel the current naming is appropriate. No strong opinions, but let's just try to keep it consistent with `getDirectedReadOptions`?

Reply:
Sounds good, done.